ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -0,0 +1,61 @@
+define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
+  'jquery_mockjax'], function(QUnit, $, ManageTaxonomies, React) {
+  'use strict';
+
+  var VocabularyComponent = ManageTaxonomies.VocabularyComponent;
+  var addTermResponse = {
+    "id": 9,
+    "slug": "aa",
+    "label": "aa",
+    "weight": 1
+  };
+
+  QUnit.test('Assert that VocabularyComponent renders properly',
+    function(assert) {
+      assert.ok(VocabularyComponent, "class object not found");
+      $.mockjax({
+        url: "/api/v1/repositories/repo/vocabularies/dummy/terms/",
+        responseText: addTermResponse
+      });
+      var vocabulary = {
+        "id": 1,
+        "slug": "difficulty",
+        "name": "difficulty",
+        "description": "easy",
+        "vocabulary_type": "f",
+        "required": false,
+        "weight": 2147483647,
+        "terms": [
+          {
+            "id": 1,
+            "slug": "ease",
+            "label": "ease",
+            "weight": 1
+          },
+          {
+            "id": 2,
+            "slug": "ease2",
+            "label": "ease2",
+            "weight": 1
+          }
+        ]
+      };
+
+      var vocabularyComponentRendered =
+        React.addons.TestUtils.renderIntoDocument(
+        <VocabularyComponent vocabulary={vocabulary} repoSlug={"repo"} />
+        );
+
+      var randomEvent = $.Event("keyup", {keyCode: 64});
+      var eventEnter = $.Event("keyup", {keyCode: 13});
+
+      assert.equal(
+        false, vocabularyComponentRendered.onEnterPress(randomEvent)
+      );
+      assert.equal(
+        true, vocabularyComponentRendered.onEnterPress(eventEnter)
+      );
+
```

Comment:
Actually, can you add an assertion here that the state has been updated with the item?